storage/kvstore.go

```diff
@@ -24,6 +24,7 @@ var (
 
 	ErrTnxIDMismatch = errors.New("storage: tnx id mismatch")
 	ErrCompacted     = errors.New("storage: required reversion has been compacted")
+	ErrUnhappenedRev = errors.New("storage: required reversion has not happened")
```

Comment:
I don't think out a good error message for it.
is `required version is a future version` good?